src/test/emulators/storage/rules/config.spec.ts

```diff
@@ -43,15 +49,20 @@ describe("Storage Rules Config", () => {
       },
       path: resolvePath,
     });
-    config.rc.applyTarget(PROJECT_ID, "storage", "main", ["bucket_1", "bucket_2"]);
-    config.rc.applyTarget(PROJECT_ID, "storage", "other", ["bucket_3"]);
+    config.rc.applyTarget(PROJECT_ID, "storage", "main", ["bucket_0", "bucket_1"]);
+    config.rc.applyTarget(PROJECT_ID, "storage", "other", ["bucket_2"]);
 
-    const result = getStorageRulesConfig(PROJECT_ID, config);
+    const result = getStorageRulesConfig(PROJECT_ID, config) as RulesConfig[];
 
     expect(result.length).to.equal(3);
-    expect(result[0]).to.eql({ resource: "bucket_1", rules: `${tmpDir}/storage_main.rules` });
-    expect(result[1]).to.eql({ resource: "bucket_2", rules: `${tmpDir}/storage_main.rules` });
-    expect(result[2]).to.eql({ resource: "bucket_3", rules: `${tmpDir}/storage_other.rules` });
+    for (let i = 0; i < result.length; i++) {
```

Comment:
small nit: prefer to keep tests as dumb (and damp) as possible

see go/tott/598

Reply:
Done.